Automattic/wp-calypso
# Anchor: Designs Gilbert and Riley use the spearhead theme

#### Changes proposed in this Pull Request

* Anchor gutenboarding designs `gilbert` and `riley` now have `spearhead` designated as their underlying theme, **allowing headstart to run**

#### Testing instructions

* Start calypso
* (optional) Sandbox public-api and add logging in sandbox that shows that headstart is running. Example: 27ef6-pb
* Visit http://calypso.localhost:3000/new?&anchor_podcast=22b6608&anchor_episode=e324a06c-3148-43a4-85d8-34c0d8222138&spotify_show_url=https%3A%2F%2Fopen.spotify.com%2Fshow%2F6HTZdaDHjqXKDE4acYffoD%3Fsi%3DEVfDYETjQCu7pasVG5D73Q and create a new site
* Specify gilbert or riley as the theme and end up in the editor
* Verify that headstart ran. Either from the logging you placed earlier, or visit a special options url ( 27ef7-pb ) and verify that the anchor_podcast option is set for that site.

Fixes 416-gh-Automattic/dotcom-manage
